### ISSUE TITLE
polars version 0.20.12 not available on pypi

### ISSUE THREAD

**Issue by poudrouxj** [pola-rs/polars#14760]
### Checks

- [X] I have checked that this issue has not already been reported.
- [X] I have confirmed this bug exists on the [latest version](https://pypi.org/project/polars/) of Polars.

### Reproducible example

```bash
pip install polars
```

### Log output

```shell
#26 11.59 ERROR: Could not find a version that satisfies the requirement polars==0.20.12 (from versions: 0.0.1, 0.7.3, 0.7.4, 0.7.5, 0.7.6b1, 0.7.6b2, 0.7.6, 0.7.7, 0.7.8, 0.7.9, 0.7.10b1, 0.7.11, 0.7.12, 0.7.13, 0.7.14, 0.7.15, 0.7.16, 0.7.17, 0.7.18, 0.7.19, 0.8.0b1, 0.8.0b2, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.8.5, 0.8.6, 0.8.8, 0.8.9, 0.8.10, 0.8.11b1, 0.8.11, 0.8.12, 0.8.13b1, 0.8.14, 0.8.15b1, 0.8.15, 0.8.16, 0.8.17, 0.8.18, 0.8.19, 0.8.20, 0.8.21b1, 0.8.21b2, 0.8.21, 0.8.22b1, 0.8.22, 0.8.23, 0.8.24, 0.8.25, 0.8.26, 0.8.27b1, 0.8.27b2, 0.8.27, 0.8.28, 0.8.29, 0.9.0b1, 0.9.0b2, 0.9.0, 0.9.1, 0.9.2, 0.9.3b1, 0.9.3, 0.9.4, 0.9.5b1, 0.9.5, 0.9.6b1, 0.9.6, 0.9.7, 0.9.8, 0.9.9, 0.9.10, 0.9.11, 0.9.12, 0.10.0, 0.10.1, 0.10.2, 0.10.3, 0.10.4, 0.10.5, 0.10.6, 0.10.7, 0.10.8, 0.10.9, 0.10.10, 0.10.11, 0.10.12, 0.10.13, 0.10.14, 0.10.15, 0.10.16, 0.10.17, 0.10.18, 0.10.19, 0.10.20, 0.10.21, 0.10.22, 0.10.23, 0.10.24, 0.10.25, 0.10.26, 0.10.27, 0.11.0, 0.11.1, 0.12.0, 0.12.1, 0.12.2, 0.12.3, 0.12.4, 0.12.5, 0.12.6, 0.12.7, 0.12.8, 0.12.9, 0.12.10, 0.12.11, 0.12.12, 0.12.13, 0.12.14, 0.12.15, 0.12.16, 0.12.17, 0.12.18, 0.12.19, 0.12.20, 0.12.21, 0.12.22, 0.12.23, 0.12.24, 0.13.0, 0.13.1, 0.13.2, 0.13.3, 0.13.4, 0.13.5, 0.13.6, 0.13.7, 0.13.8, 0.13.10, 0.13.11, 0.13.12, 0.13.13, 0.13.14, 0.13.15, 0.13.16, 0.13.17, 0.13.18, 0.13.19, 0.13.21, 0.13.22, 0.13.23, 0.13.24, 0.13.25, 0.13.29, 0.13.31, 0.13.34, 0.13.36, 0.13.37, 0.13.38, 0.13.39, 0.13.40, 0.13.44, 0.13.46, 0.13.49, 0.13.50, 0.13.51, 0.13.52, 0.13.55, 0.13.58, 0.13.59, 0.13.61, 0.13.62, 0.14.0, 0.14.1, 0.14.2, 0.14.5, 0.14.6, 0.14.8, 0.14.9, 0.14.10, 0.14.11, 0.14.14, 0.14.17, 0.14.18, 0.14.19, 0.14.21, 0.14.23, 0.14.24, 0.14.25, 0.14.26, 0.14.27, 0.14.28, 0.14.29, 0.15.1, 0.15.2, 0.15.6, 0.15.7, 0.15.8, 0.15.9, 0.15.11, 0.15.13, 0.15.14, 0.15.15, 0.15.16, 0.15.17, 0.15.18, 0.16.1, 0.16.2, 0.16.3, 0.16.4, 0.16.5, 0.16.6, 0.16.7, 0.16.8, 0.16.9, 0.16.10, 0.16.11, 0.16.12, 0.16.13, 0.16.14, 0.16.15, 0.16.16, 0.16.18, 0.17.3, 0.17.5, 0.17.9, 0.17.10, 0.17.11, 0.17.12, 0.17.13, 0.17.14, 0.17.15, 0.18.0, 0.18.2, 0.18.3, 0.18.4, 0.18.6, 0.18.7, 0.18.8, 0.18.11, 0.18.12, 0.18.13, 0.18.15, 0.19.0, 0.19.1, 0.19.2, 0.19.3, 0.19.5, 0.19.6, 0.19.7, 0.19.11, 0.19.12rc1, 0.19.12, 0.19.13rc1, 0.19.13, 0.19.14, 0.19.15, 0.19.17, 0.19.18, 0.19.19, 0.20.0, 0.20.1, 0.20.2, 0.20.3rc1, 0.20.3rc2, 0.20.3, 0.20.4, 0.20.5, 0.20.6rc1, 0.20.6, 0.20.7, 0.20.8, 0.20.9, 0.20.10)
#26 11.59 ERROR: No matching distribution found for polars==0.20.12```
```

### Issue description

Missing release on pypi makes it difficult to install

### Expected behavior

It install the latest one 0.20.12

### Installed versions

<details>

```
N/A
```

</details>

**Comment by alexander-beedie** [comment]
We had to remove it due to an urgent fix; we'll be pushing an update (0.20.13) shortly.

**Comment by poudrouxj** [comment]
> We had to remove it due to an urgent fix; we'll be pushing an update (0.20.13) shortly.

Great - thanks for a speedy reply and an awesome library !
